src/Controls/src/BindingSourceGen/GeneratorDataModels.cs

```diff
@@ -41,6 +41,26 @@ public InterceptorLocation ToInterceptorLocation()
 	}
 }
 
+public enum AccessorKind
+{
+	Field,
+	StaticField,
```

Comment:
I don't see `StaticField` used anywhere. Is that because it's not needed or because it needs to be implemented yet?

Reply:
Nice catch! Thanks! Hmm, I don't think we do support static fields in lambdas, do we? Should we?

Reply:
We don't AFAIK